platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenContainers.kt

```diff
@@ -24,12 +24,12 @@ fun yourDataContainers(
         ContainerType.ROW
     )
 
-    return chunked.mapIndexed { index, tiles ->
+    return chunked.mapIndexed { index, theseTiles ->
         when (val type = containersConfig[index % containersConfig.count()]) {
             ContainerType.LARGELEFT -> Container(
                 type = type,
-                tiles = tiles.mapIndexed { index, tileModel ->
-                    if (index == 0) {
+                tiles = theseTiles.mapIndexed { idx, tileModel ->
```

Comment:
Another option can be removing the `tiles` parameter name, as it is kinda redundant, and used only for readability.

Reply:
Nit: `idx` is a bit cryptic - `tileIndex`? But it's not important.